tests/test_api.py

```diff
@@ -584,3 +584,18 @@ def test_namespace_present():
     )
 
     assert model.namespace is namespace
+
+
+def test_export_engine():
+    """Test ExportEngine cache decorator."""
+    export_engine = ExportEngine()
+    assert export_engine.export_cache_context.get() == ExportEngine.CACHE_NOT_INITIALIZED
+
+    @ExportEngine.cache
+    def decorator_test(*args: dict, **kwargs: dict) -> None:
+        assert kwargs["export_cache"] == {}
+        kwargs["export_cache"]["a"] = "b"
+        assert kwargs["export_cache"] == {"a": "b"}
+
+    decorator_test()
```

Comment:
return and check output value - it will be called on View returning an html response